### DISCUSSION TITLE
求助求助求

### DISCUSSION THREAD

**Discussion by Guollove** [xaoxuu/hexo-theme-stellar#102] [Q&A] [answered]
![52D16E0901F896E6DE728744A6035287](https://user-images.githubusercontent.com/53716683/174243806-e6d1405b-d098-410b-b331-f490b686ab5f.jpg)

是我太笨了。研究三天啦。还有两个部分功能死活找不到
第一个。 侧边栏这么把它弄成这种小标签形式
第二个。 Sites（网站卡片标签）怎么让他分组 也有title标题 搞一天了。title标题老是冲突。

是在没办法了。只好求助了。 谢谢🙏解答。谢谢了。 谢谢。

**Comment by PluginsKers** [comment]
跟楼了，这个问题也困扰我很久了，我找不到相关文档！

**Comment by xaoxuu** [comment, marked as answer]
## 对目录树进行分组

https://xaoxuu.com/wiki/stellar/wiki-settings/#对目录树进行分组

![](https://user-images.githubusercontent.com/16400144/175819968-48aa86db-8580-432e-9e69-3461d2cdb87e.png)

## 网站卡片标签

https://xaoxuu.com/wiki/stellar/tag-plugins/#Sites（网站卡片标签）

![](https://user-images.githubusercontent.com/16400144/175819947-cc29e98e-fea5-4484-8050-775f53e5cb8d.png)
